wp-config.php

```diff
@@ -72,6 +72,12 @@
     }
     // Don't show deprecations; useful under PHP 5.5
     error_reporting(E_ALL ^ E_DEPRECATED);
+    // Force the use of a safe temp directory when in a container
+    if ( 0 === stripos( dirname( __FILE__ ), '/srv/bindings/' ) ):
```

Comment:
+1 for this change.

I think we should use the same mechanism for determining if we're "on Pantheon" we to elsewhere: looking for the presence of our signature constants. I'd check for `PANTHEON_BINDING`, which you can also use in place of `$pantheon_file_path_parts`, which will simplify this.